deploy-manage/deploy/self-managed/_snippets/cmd-line-config.md

```diff
@@ -0,0 +1,21 @@
+{{es}} loads its configuration from the following location by default:
+
+```
+{{es-conf}}{{slash}}elasticsearch.yml
+```
+
+The format of this config file is explained in [](/deploy-manage/deploy/self-managed/configure-elasticsearch.md).
+
+Any settings that can be specified in the config file can also be specified on the command line, using the `-E` syntax as follows:
+
+```sh
+.\bin\elasticsearch.bat -Ecluster.name=my_cluster -Enode.name=node_1
+```
```

Comment:
elasticsearch.bat is only useful for windows systems, and this snippet is used also in the linux doc.
Not sure if there's any way to pass specific substitutions to snippets when calling them.